### ISSUE TITLE
[CoE Starter Kit - QUESTION] Error loading the Power BI dashboard template Production_CoEDashboard_February2023.pbit

### ISSUE THREAD

**Issue by farid-jaouhari** [microsoft/coe-starter-kit#4989]
### What is your question?

We have installed the CoE twice in our tenant. The first installation was carried out on a production Power Platform environment, but the default language was set to Japanese.
After the installation, we were able to load the Power BI dashboard template "Production_CoEDashboard_February2023.pbit" and publish the report.
However, since many elements were still appearing in Japanese, we no longer wish to use this environment.
The second installation was carried out on another production Power Platform environment, and the default language was set to English.
However, after the installation, we have encountered issues loading the Power BI dashboard template "Production_CoEDashboard_February2023.pbit".
When we enter the OrgURL of the English environment, we receive below error messages:

OrgUrl
Loading blocked by failures with other queries.

Environment, App, Flow, Maker and all others
Access to the resource is forbidden.

NonProdWords
Loading blocked by failures with other queries.

Is there anything we can do to resolve the issue? It appears that the flows are functioning correctly, as we can see data in the tables of the English environment.
![OrgURL](https://user-images.githubusercontent.com/45904045/225260999-39aed564-b007-44cd-a9ab-e09eb927d3a6.png)
![OrgURL Load Error](https://user-images.githubusercontent.com/45904045/225261032-04884001-6967-40fd-8401-b535a034050a.png)
![OrgURL Load Error part 2](https://user-images.githubusercontent.com/45904045/225261052-7b74d23b-2ac5-4698-a5c0-114b1cc9f476.png)

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

February 2023

**Comment by Jenefer-Monroe** [comment]
It sounds like you do not have access to the environment. 
Go to the product admin pages https://admin.powerplatform.microsoft.com/environments
Find your environment > Settings > Users > Find your user > See what Roles you have
![image](https://user-images.githubusercontent.com/89584709/225278501-4b155167-0c0a-4eb9-80d4-04ef31d372d2.png)
![image](https://user-images.githubusercontent.com/89584709/225278614-c600ab6b-b712-48d7-9d71-bd5383fd9c25.png)
![image](https://user-images.githubusercontent.com/89584709/225279382-f63c462f-7827-49e4-936f-01d92b7ab7fc.png)

**Comment by farid-jaouhari** [comment]
Dear Jenefer,

Thank you for the prompt response.
In fact, your are right, I forgot to assign the "system administrator" role to my account.
I have done so now and I am able to load the data, with the exception of the "App" data, for which I am receiving an error message.
I am aware that this is beyond the scope of the Center of Excellence (CoE), but when I click on "View errors," the screen closes and I am returned to the report.
Is there a way for me to determine what the error is?
See screenshot:
![AppLoadError](https://user-images.githubusercontent.com/45904045/225295067-fd834d9d-34b9-4b63-9e84-902a0cffa155.png)

**Comment by Jenefer-Monroe** [comment]
I have seen this occur once in a while when a row is malformatted for some reason, ex missing its environment GUID. Running the following clean up flow from the Core solution typically resolves: **_CLEANUP - Admin | Sync Template v3 (Delete Bad Data)_**

**Comment by farid-jaouhari** [comment]
Thanks for your answer.
I can see that the flow "CLEANUP - Admin | Sync Template v3 (Delete Bad Data)" was previously run according to the schedule, and I have also manually run it myself. However, when I try to open the Power BI dashboard in Power BI Desktop, I am still encountering the same error.

Could it be related to the issues that the CoE is currently reporting via email?

![CoE_Email](https://user-images.githubusercontent.com/45904045/225630595-b9e3a509-b173-4cfc-a1d1-cbb56ac3f7a5.png)
![Flow1](https://user-images.githubusercontent.com/45904045/225630623-75124322-af13-4dec-89eb-dc6da3af0605.png)
![Flow2](https://user-images.githubusercontent.com/45904045/225630638-c6ec4b71-5b39-425c-9a79-04e324023b7a.png)
![Flow3](https://user-images.githubusercontent.com/45904045/225630673-8f073824-11e1-493c-a952-2099ac08aa2f.png)

**Comment by Jenefer-Monroe** [comment]
No those would be unrelated. 
Can you click on View errors and share what you see ther.e 
![image](https://user-images.githubusercontent.com/89584709/225631901-bd0f273b-cf43-42a2-a41b-d34af61e90d0.png)

**Comment by farid-jaouhari** [comment]
When I click on View errors, I get:

![Err1](https://user-images.githubusercontent.com/45904045/225634240-41ef4c41-9ffc-42ca-a9f6-257503fc08e8.png)

And then Power Query Editor opens:

![Err2](https://user-images.githubusercontent.com/45904045/225634335-d9df4879-dbcb-4486-af23-f0b6b7c0ebe1.png)

**Additionally**, if you can help, I am also interested in learning more about the errors that I am receiving via email and how to resolve them as well.

**Comment by Jenefer-Monroe** [comment]
In order for us to track issues easily we would like to keep each issue topic specific, so please post issues for the two flows you have failing. 

Please note that failures like the one you show in your picture though are caught/expected failures. 
We use these calls to see if the object is new to the inventory, and if it is, this call to “get it” will fail. This failure is caught and the “isnew” variable is set to true. 
![image](https://user-images.githubusercontent.com/89584709/225679681-01467de5-1a1c-4c93-bf57-71296f5ea83b.png)

In order to debug you need to find the terminating failure in the flow. Here for example the call to “forced failure” failed, and was not caught, so the flow terminated here. That is the spot you need to find in your failure in order to debug. 
![image](https://user-images.githubusercontent.com/89584709/225681214-a5549790-32e9-482e-b427-0c346aa50684.png)

**Comment by farid-jaouhari** [comment]
Thank you for your support on this matter.
I agree, it would be best for me to open a new issue topic regarding the CoE Emails with the three failing flows.
I mentioned this because I thought it might be the reason why I am receiving an error in Power BI.
I have shared the errors I am encountering in the Power BI Desktop Client. Are you familiar with this issue? Is there something I need to change in order to resolve it?

**Comment by Jenefer-Monroe** [comment]
closing out as no further action for starter kit team
